content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,186 @@
+============
+Work Entries
+============
+
+The :guilabel:`Work Entries` dashboard, which can be found by going to :menuselection:`Payroll
+--> Work Entries --> Work Entries`, provides a visual overview of the individual time sheets of
+every employee, with each day split into a morning shift and an afternoon shift.
+
+.. image:: work_entries/work-entries-overview.png
+   :align: center
+   :alt: Work Entries dashboard view showing all employee's work entries.
+
+To change the view so that only the entries for a single day, week, or month are shown, click on
+either :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month`.
+
+.. image:: work_entries/work-entry-date.png
+   :align: center
+   :alt: Select the date format to view, either day, week, or month.
+
+Use the :guilabel:`arrow icons` on the left and right side of the :guilabel:`Today` button to adjust
+the displayed dates. The arrows will adjust the date based on the type of time selected. For
+example, if month is selected, the arrows will move one month with each click of the arrow. If week
+or day is selected, the time will move by either a week or a day for each click of the arrow,
+respectively.
+
+Adding a new work entry
+=======================
+
+If a work entry is missing and needs to be added, such as  sick time or time off, click
+:guilabel:`Add` to create a new work entry. A pop-up will appear, with several fields to fill in.
+
+Enter the :guilabel:`Work Entry Name`, such as 'Sick Time' or any other short description. Select
+the :guilabel:`Employee` and the :guilabel:`Work Entry Type` from the respective drop-down lists.
+
+.. image:: work_entries/create.png
```

Comment:
ideally the screenshot of this form is filled out with a realistic information so it's more educational for the reader (since they can view blank forms on their own DBs which isn't very helpful 😅)